### ISSUE TITLE
beforeAll function doesn't work as it should

### ISSUE THREAD

**Issue by kamek-pf** [Sanjo/meteor-jasmine#262]
Hey, 
I found a bug with the `beforeAll()` function : 
```
fdescribe('Test', function () {

    beforeAll(function () {
        console.log('Hey there !');
    });

    it('number #1', function () {
        expect(true).toBe(true);
    });

    it('number #2', function () {
        expect(true).toBe(true);
    });

    it('number #3', function () {
        expect(true).toBe(true);
    });

    it('number #4', function () {
        expect(true).toBe(true);
    });
});
```

This will output `Hey there !` 5 times (which is weird considering there are only 4 tests) instead of one as expected. Using `beforeEach()` outputs the message 4 times, as it should.
I'm not sure if it's an issue with meteor-jasmine or if this happens with vanilla Jasmine as well.

**Comment by Sanjo** [comment]
Thanks for reporting. I think this is a Jasmine issue that may already be fixed in the latest version. I will test.

**Comment by kamek-pf** [comment]
Coworker tested Jasmine 2.3 with Node, says it works fine, you may want to look for something wrong in this package.

**Comment by Sanjo** [comment]
Ok. sanjo:jasmine still uses Jasmine 2.1.

**Comment by Sanjo** [comment]
This has been fixed in Jasmine 2.3. I will upgrade now. https://github.com/jasmine/jasmine/issues/773

**Comment by kamek-pf** [comment]
While you're at it, looks like `afterAll()` has the exact same behavior :p

**Comment by Sanjo** [comment]
Should be fixed with 0.17.0. https://github.com/Sanjo/meteor-jasmine/commit/2f5b3d02ed2dbd029fa39bfb2b2b5c48e8cb3685
